saleor/graphql/views.py

```diff
@@ -212,6 +212,16 @@ def parse_query(
         except (ValueError, GraphQLSyntaxError) as e:
             return None, ExecutionResult(errors=[e], invalid=True)
 
+    def check_if_query_contains_only_schema(self, document: GraphQLDocument):
+        for definition in document.document_ast.definitions:
+            selections = definition.selection_set.selections
+            if len(selections) > 1:
```

Comment:
If you revert the condition, you will reduce the indentations but it is up to you